libs/generic-view/ui/src/lib/predefined/import-contacts/import-contacts-progress.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React, { FunctionComponent, useEffect, useState } from "react"
+import styled from "styled-components"
+import { IconType } from "generic-view/utils"
+import { ProgressBar } from "../../interactive/progress-bar/progress-bar"
+import { ModalTitleIcon } from "../../interactive/modal"
+import { defineMessages } from "react-intl"
+import { intl } from "Core/__deprecated__/renderer/utils/intl"
+
+const messages = defineMessages({
+  title: {
+    id: "module.genericViews.importContacts.progress.title",
+  },
+  description: {
+    id: "module.genericViews.importContacts.progress.description",
+  },
+  progressDetails: {
+    id: "module.genericViews.importContacts.progress.progressDetails",
+  },
+})
+
+interface Props {
+  onFinish: VoidFunction
+}
+
+export const ImportContactsProgress: FunctionComponent<Props> = ({
+  onFinish,
+}) => {
+  const [progress, setProgress] = useState(0)
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setProgress((prev) => Math.min(prev + 15, 100))
+    }, 500)
```

Comment:
⚪ (question)
Is this just a placeholder progress until integration with the internal source? (mock)

Reply:
Yes, exactly